src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -717,7 +717,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
         private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Automation_ErrorRecord()
         {
             yield return new FormatViewDefinition("ErrorInstance",
-                CustomControl.Create(outOfBand: true)
+                CustomControl.Create(outOfBand: true, width: Int32.MaxValue)
                     .StartEntry()
```

Comment:
Only with the third attempt, I started to understand how the formating system works. Although it could be a hoax. 😄